modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -113,6 +136,22 @@ async function assertTableRowsCount(page, rowsCount) {
 	});
 }
 
+async function createDataSetsSequentually(fnPromiseArr) {
```

Comment:
Need a helper to ensure that the Data Sets were created in certain order and cover sorting by creationDate (default one)

Reply:
Is there a difference between this and a bunch of `await ... createDataSet`s? This looks like it works, it's just that pushing all promises in array, and then looping over that array, seems like an unnecessary step to me. But if you think it improves code oragnization, :+1: 

The only nit I'd have here is naming. I see two good way to do it:
- generic naming, something like `function resolvePromises(promises)`
- DS specific naming, something like `function createDataSets(promises)`

Reply:
I'll revert back to a bunch of `await`s. It will be clear and follows our tests pattern. 
I was only trying to ensure correct creation order until I've found the missing sorting parameters in the request :shrug: